{"name": "test_create_user_with_invalid_values_002", "status": "passed", "description": "create_user_with_invalid_values", "start": 1662026051627, "stop": 1662026053703, "uuid": "f92f73ba-6b4d-4932-83f3-f04c33e3d5d1", "historyId": "aa5d150ce42e2398a1ade11625555ecd", "testCaseId": "7fbb83514f4b26defe89cb9ec0ae36bc", "fullName": "Test_case.Test_Create_User_API#test_create_user_with_invalid_values_002", "labels": [{"name": "severity", "value": "Critical"}, {"name": "parentSuite", "value": "Test_case"}, {"name": "suite", "value": "Test_Create_User_API"}, {"name": "host", "value": "DESKTOP-KBHEFTS"}, {"name": "thread", "value": "17708-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_case.Test_Create_User_API"}]}